### PR TITLE
Add newspost for playtest-20150408

### PR DESCRIPTION
**Pull request by obrakmann**

### CODE REVIEW

File: content/news/playtest-20150408.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+We have finalized the balancing for the Red Alert mod, so it's time for another playtest!
+
+The changes this time around include:
+
+  - More balancing adjustments in Red Alert. Please see [the changelog](http://changelog.openra.net) for a detailed list of changes.
+  - Fixed enemies erroneously getting an "(Ally)" suffix in chat messages.
+  - Password protected games will be indicated as such in the server browser even when the game has already started.
```
Comment (pchote) [inline content/news/playtest-20150408.md]: You could shorten this to "Games in progress that are password protected are now indicated as such in the server browser." so that it doesn't wrap to a second line.

File: content/news/playtest-20150408.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  - Fixed helicopters trying to shoot at targets directly below them, thus wasting ammo.
```
Comment (pchote) [inline content/news/playtest-20150408.md]: "Fixed helicopters wasting ammo by trying to shoot at targets directly below them." reads a little better IMO.

File: content/news/playtest-20150408.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+More details and links to pull request can be found on our [changelog page](http://changelog.openra.net).
+
+You can find the installers for a variety of supported operating systems in our [download section](/download/).
+
+This will likely be your last chance to give feedback and contribute to this release, so if you find bugs, encounter any problems are have other suggestions, don't hesitate and report them at http://bugs.openra.net!
```
Comment (pchote) [inline content/news/playtest-20150408.md]: The bug tracker url should be a link.

**Comment by pchote** [comment]
Looks good otherwise, so :+1: after addressing the above nits.

**Comment by Mailaender** [comment]
My social media account maintenance experience showed me this news post NEEDS MOAR PICTURES so here you get one:

image

**Comment by obrakmann** [comment]
Fixed language, added a screenshot (from d2k instead of ra, for better visibility of the red font).

**Comment by pchote** [comment]
That font color contrast is RA is unusably horrible.

**Comment by pchote** [comment]
If we wanted a more exciting screenshot we could include one from TS showing the newish building modules.

**Comment by obrakmann** [comment]
That's for incompatible games only, though. Compatible protected games are readable, so it's not too critical, IMO. Still bad, of course.

**Comment by obrakmann** [comment]
Updated with playtest id.

I wouldn't mind adding another ts teaser in the form of [this small video](https://seacloud.cc/d/5d1adbebce/files/?p=/openra/ts-upgrades.webm), but it would need to go on our youtube channel first.

**Comment by Mailaender** [comment]
Why not use the GIF from https://github.com/OpenRA/OpenRA/pull/7812?

upgrades

**Comment by obrakmann** [comment]
Because I want to avoid adding such (relatively) huge images to our repo, and I think it is about time for us to start using youtube to show content.  People would *hear* what TS sounds like in OpenRA for the first time :)

Same thing with the weather previews, should we do one: as chrisf correctly said, the illusion only feels real when you add sound to the mix.

**Comment by Mailaender** [comment]
YouTube is also better for social media. I tried GIF at Facebook and it did not work at all.

**Comment by obrakmann** [comment]
Updated with video

File: content/news/playtest-20150408.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+A note on map compatibility: already with the first playtest a new map format was introduced which is not backward-compatible with last October's release-20141029. This means that if you have downloaded custom maps from [the Resource Center](http://resource.openra.net) or elsewhere, these maps will have been automatically updated to the new map format once you have run this or one of the previous playtests. If you later go back to the release version, it will not be able to parse the new map format and crash. You can avoid the problem by either deleting the contents of your `map` folder, or by making backups of the `map` folder and restoring that when you switch back to release-20141029.
+
+<div style="text-align:center" markdown="1">
+![Server browser with password protected games](/images/news/20150408-d2k-serverbrowser.png)
+
+Protected games in the server browser.
+
+<iframe width="600" height="450" src="https://www.youtube-nocookie.com/embed/SBU8XYuFk0M?rel=0" frameborder="0" allowfullscreen></iframe>
+
+Another sneak peak at our in-progress Tiberian Sun mod: How to deploy upgrades
```
Comment (ScottNZ) [inline content/news/playtest-20150408.md]: sneak peek*

**Comment by Mailaender** [comment]
Updated https://build.opensuse.org/package/show/games:openra/playtest
